test/fixtures/operatingsystems.yml

```diff
@@ -65,7 +65,7 @@ no_hosts_os:
   title: 'NoHosts 1.1'
 
 vrp5:
-  name: Huawei_VRP
+  name: VRP_Huawei
```

Comment:
This is exactly a case where we will say the OS is not recognized ... It's good imho, but I can't judge what users can be using now.

Reply:
@UXabre you added this - could you take a look if this is really correct?